misc/msvc/q3_ui.def

```diff
@@ -1,3 +0,0 @@
-EXPORTS
-	vmMain
-	dllEntry
```

Comment:
Same as the other def file. What is this used for? Could this be a required file for building the QVMs? If so, I don't want to delete it.